org.eclipse.xtext.builder.tests/src/org/eclipse/xtext/builder/impl/BuilderPerformanceTest.java

```diff
@@ -96,151 +94,152 @@
-	
-	@Override
+
+	private boolean doPrint = false;
+
+	@Inject
+	private PluginProjectFactory projectFactory;
+
 	@Before
-	public void setUp() throws Exception {
+	public void enableStopwatches() throws Exception {
 		Stopwatches.setEnabled(true);
-		super.setUp();
 	}
-	
-	@Override
+
 	@After
-	public void tearDown() throws Exception {
-		super.tearDown();
+	public void disableStopwatches() throws Exception {
 		printAndClearStopwatchData();
 		Stopwatches.setEnabled(false);
 	}
-	
-	@Test public void testSecondBuildRunIsMuchFaster() throws Exception {
+
+	@Test
+	public void testSecondBuildRunIsMuchFaster() throws Exception {
 		final String project1name = "project1";
 		IJavaProject javaProject1 = createJavaProject(project1name);
 		addJarToClasspath(javaProject1, copyAndGetXtendExampleJar(javaProject1));
-		
-		System.out.println("Clean Java first - "+cleanBuildTakes());
-		System.out.println("Clean Java second - "+cleanBuildTakes());
-		
+
+		if (doPrint) {
+			System.out.println("Clean Java first - " + cleanBuildTakes());
+			System.out.println("Clean Java second - " + cleanBuildTakes());
+		}
 		toggleNature(javaProject1);
-		
-		System.out.println("Clean Xtext first - "+cleanBuildTakes());
-//		final Controller controller = new Controller();
-//		controller.startCPUProfiling(ProfilingModes.CPU_TRACING, null, "");
-		System.out.println("Clean Xtext second - "+cleanBuildTakes());
-		System.out.println("Clean Xtext third - "+cleanBuildTakes());
-//		controller.captureSnapshot(ProfilingModes.SNAPSHOT_WITHOUT_HEAP);
-//		controller.stopCPUProfiling();
+		if (doPrint) {
+			System.out.println("Clean Xtext first - " + cleanBuildTakes());
+//			final Controller controller = new Controller();
+//			controller.startCPUProfiling(ProfilingModes.CPU_TRACING, null, "");
+			System.out.println("Clean Xtext second - " + cleanBuildTakes());
+			System.out.println("Clean Xtext third - " + cleanBuildTakes());
+//			controller.captureSnapshot(ProfilingModes.SNAPSHOT_WITHOUT_HEAP);
+//			controller.stopCPUProfiling();
+		}
 	}
 
-	@Test public void testBuildWithJarsContainingIndexedResources() throws Exception {
+	@Test
+	public void testBuildWithJarsContainingIndexedResources() throws Exception {
 		final String project1name = "project1";
 		// create a project with a bunch of jars
-		IProject project1 = createPluginProject(project1name, 
-				"org.eclipse.core.runtime", 
-				"org.eclipse.core.jobs", 
-				"org.eclipse.core.variables", 
-				"org.eclipse.debug.core", 
-				"org.eclipse.debug.ui", 
-				"org.eclipse.emf.edit", 
-				"org.eclipse.emf.ecore", 
-				"org.eclipse.emf.common", 
-				"org.eclipse.emf.ecore.xmi", 
-				"org.antlr.generator", 
-				"org.eclipse.jdt.ui", 
-				"org.eclipse.emf.mwe.core", 
-				"org.eclipse.jdt.core", 
-				"org.eclipse.xtext.xbase.lib", 
-				"org.eclipse.xtend.lib");
+		IProject project1 = createPluginProject(project1name, "org.eclipse.core.runtime", "org.eclipse.core.jobs",
```

Comment:
reformatting intended?

Reply:
Hit format by accident inbetween. Will fix it